### PR TITLE
Features/UI

### PR DESCRIPTION
**Pull request by zilu0907**
1. Add "hide repls" mode. When user is editing the code in the editor, they may need more space, and can hide repls for a while, and display it when needed.
2. Add styles for buttons.
3. Change background color and borders of "editor" div and "interactions" div
4. Increase the width of scroll bar to 15px.
5. Increase the height of the editor to 400px.
image

### CODE REVIEW

**Review by shanliyu — CHANGES_REQUESTED** [review]
Seems like .DS_Store is still being committed here. Also this theme dropdown is already fixed  in the [hotfix](https://github.com/shanliyu/chocopy-wasm-compiler/commit/2ba53321710a47b2356dd64ea115d62ccbffb320) I pushed this morning. Can you try to merge with front-end and then make this PR?

File: style.scss [hunk 1]
```diff
@@ -83,6 +85,9 @@
   overflow-y: scroll;
```
Comment (shanliyu) [inline style.scss:85]: Can we change height to percentage(e.g. 80%) rather than fixed pixal? This is related to what I brought up on slack

Comment (zilu0907) [reply style.scss:85]: sorry which height?

Comment (zilu0907) [reply style.scss:85]: Also, I've deleted .DS_Store

Comment (zilu0907) [reply style.scss:85]: Oh I know what you mean now

Comment (shanliyu) [reply style.scss:85]: Sorry I meant the height for interactions, it is set to 600px

Comment (shanliyu) [reply style.scss:85]: Line [here](https://github.com/shanliyu/chocopy-wasm-compiler/blob/4810b192f773d5287993bba08ab9462f30b0cea6/style.scss#L84)

Comment (zilu0907) [reply style.scss:85]: It seems impossible to let height be the percentage of the screen height in css. So I still used fixed pixel, but added a breakpoint at height=900px for now. If it still doesn't look good on your device, we may further adjust it.

Comment (shanliyu) [reply style.scss:85]: Got it. Thanks for looking into this!

**Comment by shanliyu** [comment]
LGTM

**Pull request merged**